<launch>

  <include file="$(find raspicam_node)/launch/camerav2_1280x960.launch"/>
  <node name="image_converter" pkg="image_transport" type="republish" args="compressed in:='/raspicam_node/image' raw out:='/raspicam_node/image_raw'"/>

<!--<arg name="marker_size" default="10.5" /> drone-->
	<arg name="marker_size" default="9.6" />
<!--<arg name="marker_size" default="18.747" /> big tb-->
	<arg name="max_new_marker_error" default="0.08" />
	<arg name="max_track_error" default="0.2" />
	<arg name="cam_image_topic" default="/raspicam_node/image_raw" />
	<arg name="cam_info_topic" default="/raspicam_node/camera_info" />
	<arg name="output_frame" default="/raspicam" />
  <node name="feature_tag_tracker" pkg="ar_track_alvar" type="individualMarkersNoKinect" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame)" />
</launch>
